stable/tomcat/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 name: tomcat
 description: Deploy a basic tomcat application server with sidecar as web archive container
 icon: http://tomcat.apache.org/res/images/tomcat.png
-version: 0.2.0
-appVersion: 7.0
+version: 0.3.0
+appVersion: "7.0"
```

Comment:
why are you quoting the appVersion ?

Reply:
If you take a look at the generated [index.yaml file](https://kubernetes-charts.storage.googleapis.com/index.yaml), the unquoted `7.0` (a decimal number) will be cut to `"7"` unexpectedly.

![image](https://user-images.githubusercontent.com/4861515/63135763-98a59700-c001-11e9-8c8c-4e62dc11d8f5.png)